### ISSUE TITLE
automatic inversion of x axis with pandas dataframe read from a csv

### ISSUE THREAD

**Issue by kinyatoride** [lukelbd/proplot#191]
### Description

The x axis of histogram is automatically inverted when plotted with a pandas dataframe read from a csv file. It's fine when the data is not from a csv file as far as I observed.

### Steps to reproduce

```python
import numpy as np
import pandas as pd
import proplot as plot

rng = np.random.default_rng()
x = rng.standard_normal(500)
x = pd.DataFrame(x, columns=['col'])

f = 'test.csv'
x.to_csv(f)
x1 = pd.read_csv(f, index_col=0)
fig, ax = plot.subplots()
ax.hist(x1.col.values)
plot.show()
```

**Expected behavior**: [What you expected to happen]

**Actual behavior**: [What actually happened]
**![Screen Shot 2020-06-16 at 10 20 21](https://user-images.githubusercontent.com/61028484/84806735-fd7fa400-afba-11ea-84e1-7257db68549f.png)**

### Equivalent steps in matplotlib
```python
import matplotlib.pyplot as plt
fig, ax = plt.subplots()
ax.hist(x1.col.values)
```

### Proplot version

0.6.4

**Comment by lukelbd** [comment]
Thanks for the report, this was a very simple fix: 1290536

**Comment by scottstanie** [comment]
<img width="752" alt="image" src="https://user-images.githubusercontent.com/8291800/110735783-0f137680-81f0-11eb-91e0-651241b93f69.png">

This seems to happen when I specify the number of bins?

But doesn't without that...

<img width="651" alt="image" src="https://user-images.githubusercontent.com/8291800/110735818-218db000-81f0-11eb-8f28-e153c69ec3fa.png">
